File: app/src/main/java/com/work/lazxy/writeaway/event/EventImportComplete.java
package com.work.lazxy.writeaway.event;

/**
 * Created by Lazxy on 2018/7/22.
 */
public class EventImportComplete {
    public boolean mIsSuccessful;
    public String mErrorMessage;
    public int mCurrentIndex;
    public int mTaskSum;

    public EventImportComplete(int index, int taskSum, boolean isSuccessful) {
        mIsSuccessful = isSuccessful;
        mCurrentIndex = index;
        mTaskSum = taskSum;
    }

    public EventImportComplete(boolean isSuccessful, String errorMessage) {
        mIsSuccessful = isSuccessful;
        mErrorMessage = errorMessage;
    }
}

File: app/src/main/java/com/work/lazxy/writeaway/event/EventImportComplete.kt
package com.work.lazxy.writeaway.event

/**
 * Created by Lazxy on 2018/7/22.
 */
class EventImportComplete {
    var mIsSuccessful: Boolean
    var mErrorMessage: String = ""
    var mCurrentIndex = 0
    var mTaskSum = 0

    constructor(index: Int, taskSum: Int, isSuccessful: Boolean) {
        mIsSuccessful = isSuccessful
        mCurrentIndex = index
        mTaskSum = taskSum
    }

    constructor(isSuccessful: Boolean, errorMessage: String) {
        mIsSuccessful = isSuccessful
        mErrorMessage = errorMessage
    }
}